phoenix-core/src/main/java/org/apache/phoenix/schema/transform/Transform.java

```diff
@@ -488,7 +488,11 @@ public static void doCutover(PhoenixConnection connection, SystemTransformRecord
         connection.setAutoCommit(false);
         List<TableInfo> viewsToUpdateCache = new ArrayList<>();
         try {
-            connection.createStatement().execute(changeTable);
+            try {
+                connection.prepareStatement(changeTable).execute();
+            } catch (SQLException e) {
+                throw new SQLException("Error during cutover via change table");
```

Comment:
This is overriding exact error reported from the query better need not handle SQLException and through the same Exception.